### PR TITLE
translation of TreeSelect

### PR DESCRIPTION
**Pull request by ddcat1115**
#1471

### CODE REVIEW

**Comment by visvadw** [comment]
@ddcat1115, how should I properly to the review? Currently, I don't have push access to `ddcat1115:tree-select-translation` branch. The demos are great, I just want to do some small adjustments on `index.en-US.md`.

**Comment by visvadw** [comment]
Meanwhile, I've added some adjustments to the english versions of `Transfer`.
Sorry for providing small (one file only) PRs, but for now I'm doing the review directly on the GitHub Web interface.

**Comment by benjycui** [comment]
@visvadw Go to `Files changed`, and hover on the line which you want to comment, a blue button will appear, then click it. Just comment on GitHub, and then @ddcat1115 will update her PR, thanks.

File: components/tree-select/demo/treeData.md [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
+  en-US: Generate form tree data
 ---
 
+## zh-CN
+
 使用 `treeData` 把 JSON 数据直接生成树结构。
 
+## en-US
+
+By using `treeData`, you can build a tree structure from JSON data directly.
```
Comment (visvadw) [inline components/tree-select/demo/treeData.md]: The tree structure can be populated if the `treeData` property is provided.

File: components/tree-select/index.en-US.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+category: Components
+type: Form Controls
+title: TreeSelect
+---
+
+Tree selection control.
+
+## When to use
+
+`TreeSelect` is a selection control like `Select`, which has a tree shaped data structure, such as corporate hierarchy、 disciplinary system、 directory structure and etc.
```
Comment (visvadw) [inline components/tree-select/index.en-US.md]: `TreeSelect` is similar with `Select`, but the values are provided in a tree like structure.
Any data whose entries are defined in a hierarchical manner is fit to use this control. Examples of such case may include a corporate hierarchy, a directory structure, and so on.

Comment (visvadw) [inline components/tree-select/index.en-US.md]: For the rest of tree-select / index.en-US.md, I would provide some adjustments after the commit to master as there are many entries I would like to change: for example, I would make all the description start with capitals. 😄

**Comment by benjycui** [comment]
这。。应该 rebase 才对？

**Comment by ddcat1115** [comment]
rebase 啦

**Pull request merged**